grpc/src/test/java/com/linecorp/armeria/common/grpc/GrpcExceptionHandlerFunctionBuilderTest.java

```diff
@@ -111,21 +111,25 @@ void mapStatus() {
                 GrpcExceptionHandlerFunction
                         .builder()
                         .on(A2Exception.class, (ctx, throwable, metadata) -> Status.PERMISSION_DENIED)
+                        .on(A1Exception.class, (ctx1, cause, metadata) -> Status.DEADLINE_EXCEEDED)
                         .build();
 
         for (Throwable ex : ImmutableList.of(new A2Exception(), new A3Exception())) {
-            final Status status = Status.UNKNOWN.withCause(ex);
             final Metadata metadata = new Metadata();
-            final Status newStatus = GrpcStatus.fromExceptionHandler(exceptionHandler, ctx, status, metadata);
+            final Status newStatus = exceptionHandler.orElse(GrpcExceptionHandlerFunction.ofDefault())
+                                                     .apply(ctx, ex, metadata);
```

Comment:
It seems like we don't need the default exception handler for this test.
```suggestion
            final Status newStatus = exceptionHandler.apply(ctx, ex, metadata);
```

Reply:
Applied feedback, Thanks.
I've checked that test is working fine after change.